### PR TITLE
WRO-12406: Fixed not inserting title into the output HTML

### PR DESCRIPTION
**Pull request by hong6316**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
After html-webpack-plugin v5, the plugin is no longer inserting title via modifying `plugin.options.title` on `beforeAssetTagGeneration` hook.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Insert title tag to `htmlPluginData.headTags` in `afterTemplateExecution` hook.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRO-12406

### Comments
Enact-DCO-1.0-Signed-off-by: Taeyoung Hong (taeyoung.hong@lge.com)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# unreleased
+
+* `WebOSMetaPlugin`: Fixed not inserting title into the output HTML.
```
Comment (MikyungKim) [inline CHANGELOG.md:3]:
```suggestion
* `WebOSMetaPlugin`: Fixed to insert a title into the output HTML.
```

File: plugins/WebOSMetaPlugin/index.js [hunk 2]
```diff
@@ -174,16 +174,17 @@ class WebOSMetaPlugin {
 			// Hook into html-webpack-plugin to dynamically set page title
 			if (this.options.htmlPlugin) {
 				const htmlPluginHooks = this.options.htmlPlugin.getHooks(compilation);
-				htmlPluginHooks.beforeAssetTagGeneration.tapAsync('WebOSMetaPlugin', (htmlPluginData, callback) => {
+				htmlPluginHooks.beforeEmit.tapAsync('WebOSMetaPlugin', (htmlPluginData, callback) => {
```
Comment (MikyungKim) [inline plugins/WebOSMetaPlugin/index.js:177]:
```suggestion
				htmlPluginHooks.afterTemplateExecution.tapAsync('WebOSMetaPlugin', (htmlPluginData, callback) => {
```

File: plugins/WebOSMetaPlugin/index.js [hunk 3]
```diff
@@ -174,16 +174,17 @@ class WebOSMetaPlugin {
...
-								htmlPluginData.plugin.options.title === 'Webpack App')
-						) {
-							htmlPluginData.plugin.options.title = appinfo.obj.title;
+						let titleSubstring = htmlPluginData.html.match(/<title>.*<\/title>/g)[0];
+						let title = titleSubstring.substring(7, titleSubstring.length - 8);
+						if (appinfo.obj.title && (!title || title === 'Webpack App')) {
+							htmlPluginData.html = htmlPluginData.html.replace(
+								titleSubstring,
+								'<title>' + appinfo.obj.title + '</title>'
+							);
```
Comment (MikyungKim) [inline plugins/WebOSMetaPlugin/index.js:187]:
```suggestion
						if (appinfo.obj.title && (!title || title === 'Webpack App')) {
							const titleTag = {
								tagName: 'title',
								closeTag: true,
								innerHTML: appinfo.obj.title
							};
							htmlPluginData.html = htmlPluginData.html.replace(/<title>.*<\/title>/g, '');
							htmlPluginData.headTags.unshift(titleTag);
```

**Comment by MikyungKim** [comment]
Please resolve the merge conflict.

File: plugins/WebOSMetaPlugin/index.js (hunk 2 above)
Comment (hong6316) [reply plugins/WebOSMetaPlugin/index.js]: OK

File: plugins/WebOSMetaPlugin/index.js (hunk 3 above)
Comment (hong6316) [reply plugins/WebOSMetaPlugin/index.js]: OK

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**